cabi/cpu-cabi.cc

```diff
@@ -31,11 +31,14 @@ BOCHSAPI void cpu_new(unsigned id) {
     BX_CPU(id)->sanity_checks();
 
     BX_INSTR_INITIALIZE(id);
+
```

Comment:
I assume there’s no value in doing this before the sanity checks, otherwise they would have already failed?